test/run-gen-spec-js.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+#
+# Copyright 2016 WebAssembly Community Group participants
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import argparse
+import os
+import subprocess
+import sys
+
+import find_exe
+import utils
+from utils import Error
+
+SCRIPT_DIR = os.path.dirname(os.path.abspath(__file__))
+GEN_SPEC_JS_PY = os.path.join(SCRIPT_DIR, 'gen-spec-js.py')
+
+
+def main(args):
+  parser = argparse.ArgumentParser()
+  parser.add_argument('-o', '--out-dir', metavar='PATH',
+                      help='output directory for files.')
+  parser.add_argument('--wast2wasm', metavar='PATH',
+                      help='override wast2wasm executable.')
+  parser.add_argument('--wasm2wast', metavar='PATH',
+                      help='override wasm2wast executable.')
+  parser.add_argument('--js', metavar='PATH',
+                      help='specify JavaScript compiler to use.')
```

Comment:
Would it be better to called this "JavaScript interpreter" or "JavaScript runtime".   How about "the name of a JavaScript execution engine with which to run the generated JavaScript file.  If not specified the JavaScript will be written to stdout"?